.style.yapf

```diff
@@ -1,6 +1,10 @@
 [style]
 based_on_style = pep8
 
+# New directories should use a .style.yapf that does not include the following:
+column_limit = 80
+indent_width = 2

```

Comment:
Chromium discourages this everywhere in favor of pep-8 which is 79 columns and 4 indent. Pls remove this.

Reply:
I realise I copied and pasted without reading. Done.